vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -163,14 +164,19 @@ func verifyAddValidatorTx(
 	}
 
 	// Verify the flowcheck
+	feeCalculator := fees.NewStaticCalculator(backend.Config, currentTimestamp)
```

Comment:
in next PR we'll introduce the `fees.NewDynamicFeesCalculator` to be used post E upgrade activation